### ISSUE TITLE
Breaks greenkeeperio-bot

### ISSUE THREAD

**Issue by MTRNord** [clahub/clahub#146]
Is there a way to add bot user as the owner of an project? Because at this point the Bot always says "breaks-build" because clahub gives an error

**Comment by mzagaja** [comment]
Hi @MTRNord, thanks for your question. I am not familiar with greenkeeprio bot but as a general matter CLAHub requires a GitHub user account to be the owner of a project. If you can provide additional context around what greenkeeperio is trying to do and what specifically is blocking that, we can ascertain whether this is a bug or would be a new feature and how that feature would need to work. If this is a new feature request I am happy to review and potentially accept a pull request that would enable this functionality but currently have some other features in the pipeline that would take priority.

**Comment by MTRNord** [comment]
So greenkeepio bot does update npm dependencies and generates pull requests for it. When there are Tests it changes the title. So because it can't accept the CLAHub texts it will always say "breaks build" which is not true. So my question or my feature request is to have the ability to except users by username, so that github and CLAHub things that for example "greenkeeperio-bot" signed the License of an Project and the bot is able to get true Data. :) 

TL:DR: I wanted to request an feature that CLAHub also says "test" worked even if an user didn't sign an License if the Username (or link or whatever is needed) is in an "Exception-List" defined by the owner of an Project.

**Comment by mzagaja** [comment]
Oh I see. So a quick code level fix might be to do something similar to https://github.com/clahub/clahub/pull/136. However it sounds like an ideal world we would abstract that into a list that users could populate with whatever code updating bots they decide to use (or we do it for them via a canonical list of all code updating bots). There are some legal/policy issues I'm going to dig into around this idea generally but in the meantime I'm open to reviewing suggestions as to an approach on the technical end.

**Comment by fusion94** [comment]
refactoring wont fix
